package.json

```diff
@@ -1,29 +1,31 @@
 {
   "name": "leaderboard",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "babel,config.js",
   "scripts": {
-    "build": "webpack --mode production",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "start": "webpack",
+    "dev": "webpack serve"
```

Comment:
- [ ] I have noticed that you are missing the `build` script in the `package.json` file. This build script is a tool that lets you bundle your JavaScript applications we need to ensure that it is available. Kindly ensure that it is considered by following this suggested [documentation](https://webpack.js.org/guides/getting-started/) and/or any other preferred source. In addition, verify that the `npm run start` works as intended 👍 ☺️ and ensure proper installation.